Firestore/Source/Remote/FSTRemoteStore.m

```diff
@@ -98,6 +106,9 @@ - (instancetype)initWithLocalStore:(FSTLocalStore *)localStore
  */
 @property(nonatomic, assign) FSTOnlineState watchStreamOnlineState;
 
+/** A count of consecutive failures to open the stream. */
+@property(nonatomic, assign) NSUInteger watchStreamFailures;
```

Comment:
We should stick with avoiding these types unless we specifically need to interoperate with platform APIs that require their use. We should also avoid unsigned types unless we specifically need them: failure is very hard to detect.

Why not `int`?

Reply:
Sorry, I copypasta'd NSUInteger from kMaxPendingWrites.  I fixed that and the 2 usages I added.